tools/klee-replay/klee-replay.c

```diff
@@ -75,11 +75,11 @@ static void stop_monitored(int process) {
     /* Make sure gdb doesn't talk to the user */
     close(0);
 
-    fprintf(stderr, "KLEE-REPLAY: NOTE: RUNNING GDB: ");
+    fputs("KLEE-REPLAY: NOTE: RUNNING GDB: ", stderr);
```

Comment:
There was no new line here.

Reply:
And there is no newline now?! fputs != puts

Reply:
Ha! You're supporting my point! ;)